frontend/src/features/browser-profiles/profile-browser.ts

```diff
@@ -151,6 +151,20 @@ export class ProfileBrowser extends TailwindElement {
         ),
       );
     }
+    if (changedProperties.has("browserNotAvailable")) {
+      if (this.browserNotAvailable) {
+        window.removeEventListener("beforeunload", this.onBeforeUnload);
+      } else {
+        window.addEventListener("beforeunload", this.onBeforeUnload);
+      }
+      this.dispatchEvent(
+        new CustomEvent<BrowserErrorDetail>("btrix-browser-error", {
+          detail: {
+            error: new Error(),
```

Comment:
It might be useful to have a descriptive error message here?

Reply:
Simplified / renamed this error to not have a detail at all, since it's just 'browser not available' is all we know.